src/apps/content/templates/base.html

```diff
@@ -0,0 +1,22 @@
+<!doctype html>
+<html lang="en">
+<head>
+    {% block header %}
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    {% block meta %}
+    <title>Awesome ecommerce</title>
+    {% endblock meta %}
+
+    {% endblock header %}
```

Comment:
header is body block

Reply:
<head> html block is a home for seo meta tags

<body> is home for header/footer/content and other valuable blocks

Reply:
My bad. Moved it to the body block.